pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -29,15 +29,6 @@ import (
 )
 
 const (
-	// maxAllowableNUMANodes specifies the maximum number of NUMA Nodes that
-	// the TopologyManager supports on the underlying machine.
-	//
-	// At present, having more than this number of NUMA Nodes will result in a
-	// state explosion when trying to enumerate possible NUMAAffinity masks and
-	// generate hints for them. As such, if more NUMA Nodes than this are
-	// present on a machine and the TopologyManager is enabled, an error will
-	// be returned and the TopologyManager will not be loaded.
-	maxAllowableNUMANodes = 8
```

Comment:
I'd keep this constant but renamed like `DefaultMaxAllowableNUMANodes` - if user doesn't specify a new value, this is the recommended default. The user should probably not be enabled to set less (what's the point?) but if they set more, they should IMO get a warning both in the logs and from the docs

Reply:
That make sense, thanks for the comment.

Reply:
> but if they set more, they should IMO get a warning both in the logs and from the docs

Can you tell me where this warning or log or docs would be better if they set more? thanks!